policyengine_us/parameters/gov/states/az/tax/income/exemptions/blind/amount.yaml

```diff
@@ -0,0 +1,15 @@
+description: Arizona provides an exemption of this amount per blind filer or spouse.
+metadata:
+  unit: currency-USD
+  period: year
+  label: Arizona blind exemption amount
+  reference: 
+    - title: Arizona Form 140 (2022) Line 39
+      href: https://azdor.gov/sites/default/files/2023-03/FORMS_INDIVIDUAL_2022_140-2Df.pdf#page=2
```

Comment:
i dont think I can open the form, is it just on my end?

Reply:
I think they updated links to the form. I will replace with the new one.